test/Microsoft.Framework.FileSystemGlobbing.Tests/FunctionalTests.cs

```diff
@@ -76,7 +106,37 @@ public void IncludeCaseSensitive(string root, string includePattern, string[] ex
                                                                                     "sub/sub/preprocess-source3.txt" })]
         public void ExcludeCaseSensitive(string root, string excludePattern, string[] expectedFiles)
         {
-            var matcher = new Matcher();
+            var matcher = new Matcher(StringComparison.Ordinal);
+            matcher.AddInclude("**/*.*");
+            matcher.AddExclude(excludePattern);
+
+            ExecuteAndVerify(matcher, root, expectedFiles.Select(f => root + "/" + f).ToArray());
+        }
+
+        [Theory]
+        [InlineData("src/project/compiler/preprocess/", "source.cs", new string[] { "preprocess-source1.cs",
+                                                                                    "sub/preprocess-source2.cs",
```

Comment:
These indentations will be a pain to maintain if any of the previous string lengths change. Can we just use VS default indents here? Using a `MemberData` might pretty it up.